<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page1st</class>
 <widget class="QWidget" name="Page1st">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: white;
 }</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>150</y>
     <width>401</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    background-color: #FFFDF8;
    border-radius: 35px; 
    padding: 20px; 
    margin-top: 10px;  
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>260</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color : #FFD08A;
    border-radius: 10px;
	color: white;
    outline: none; 

}

QPushButton:hover {
background-color:#FFBE70;
}</string>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_id">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>140</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #AAAAAA;
border-radius: 10px; 
padding-left: 25px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_pw">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>190</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #AAAAAA;
border-radius: 10px; 
padding-left: 25px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_login">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>50</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>23</pointsize>
      <fontweight>ExtraBold</fontweight>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFDF8;</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="margin">
     <number>-15</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_result1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>238</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFDF8;
color: #FF8A00;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_result2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>307</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFDF8;
color: #FF8A00;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>362</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/CI2.png);
background-color: none;
background-repeat: no-repeat;
background-position: center;
background-size: contain;  </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1401</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FF8A00;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_parking">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>721</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
background-repeat: no-repeat;
background-position: center;
background-size: contain;  </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>label_parking</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
